CIP-tx-serialization-deprecation-cycles/README.md

```diff
@@ -0,0 +1,116 @@
+---
+CIP: $\pi$
+Title: Transaction Serialization Deprecation Cycle
+Authors: Jared Corduan <jared.corduan@iohk.io>
+Status: Draft
+Type: Standards Track
```

Comment:
There will no longer be a `Type:` field but instead a `Category:` (https://github.com/cardano-foundation/CIPs/blob/cip-cps-rework/CIP-0001/README.md#categories) which I would think should be `Ledger`.  But I don't see that on the list of categories, nor do I see a `Core` category that I thought I read during recent drafts of CIP-0001.

@KtorZ I must have missed something about how CIPs for core features are categorised & why core features aren't on the current list of choices; what would we do in this case?

Reply:
I'd urge the respective maintainers of the various areas of the protocol to register an appropriate category for their respective areas. It is obvious that the cardano-ledger wants to engage more with the CIP process; and @JaredCorduan has been a recurring point of contact there; So simply capturing that in a document similar to CIP-0035 would be the way to go.

Reply:
https://github.com/cardano-foundation/CIPs/pull/456